kitsune/sumo/static/sumo/js/main.js

```diff
@@ -5,6 +5,29 @@ window.k = window.k || {};
 (function () {
   k.LAZY_DELAY = 500;  // delay to lazy loading scripts, in ms
   k.STATIC_URL = $('body').data('static-url');
+  k.getPathAsDict = function(propertyNames) {
+    var pathDict = {},
+      pathArray = window.location.pathname.split('/').filter(Boolean);
+
+    propertyNames.map(function(value, index) {
+      pathDict[value] = pathArray[index];
+    });
+
+    return pathDict;
+  };
+
+  k.pathStringFromDict = function(pathDict) {
```

Comment:
Since this and `getPathAsDict` are only used by new ES6 code, how do you feel about defining them directly in `kitsune/sumo/static/sumo/js/utils/urls.es6.js` instead of here and then letting `urls.es6.js` get them in the shim?